aptos-move/aptos-aggregator/src/delta_change_set.rs

```diff
@@ -540,7 +500,8 @@ mod test {
         let state_view = AggregatorStore::default();
         let delta_op = delta_add(10, 1000);
         assert_matches!(
-            delta_op.try_into_write_op(&state_view, &KEY),
+            state_view
+                .try_convert_aggregator_v2_delta_into_write_op(&KEY.clone().into(), &delta_op),
```

Comment:
Just trying to understand the design pattern. What's the benefit of moving `try_into_write_op` from delta_change_set to state_view?

Reply:
We use AggregatorResolver to create a write op, but it leaves in `aptos-aggregator` and the write op conversion was in `vm-types` which imports that. We would have a cyclic dependency if we use it as is :( I think we should think what belongs to aggregators, what is general enough we can place into types, but can be done in the future PRs